spec/messages_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "messages"
+
+describe Messages do
+
+  let(:messages) { Messages.new }
+
+  it "returns inital prompt" do
+    expect(messages.welcome_message()).to eql("Welcome to rock, paper or scissors!")
```

Comment:
These are nice straightforward tests, but remember for later when you possibly have longer messages or if to change messages when you make user experience changes, how this could be a brittle test.
You can make it less brittle by using the `includes` matcher and then only including a string of the bits you care about in the message. For instance, the welcome message it might be important that it contains "welcome" and maybe we don't care about the rest changing. You decide 😎